### DISCUSSION TITLE
Merge with fork source?

### DISCUSSION THREAD

**Discussion by da4089** [py-vobject/vobject#47] [General]
@skarim has successfully transferred ownership of the original vObject repo from Eventable (its former host company) to their own GitHub account.  This is a great effort and outcome, and in my own experience of attempting such things, no small achievement.

The question now is what to do next.

The original repo has a large collection of GitHub stars, and is the source of many forks.  While this can be viewed basically as a vanity metric, I think it goes beyond that, and conveys a level of maturity and trust in the project that is genuine, and valuable.

So, one option is to:

1. Rename py-vobject/vobject to eg. py-vobject/vobject-fork
2. Transfer skarim/vobject to py-vobject/vobject
3. Merge py-vobject/vobject-fork back into py-vobject/vobject
4. Move forward with development in py-vobject/vobject (and archive py-vobject/vobject-fork)

There'd be some mess to clean up with issues, PRs, discussions, etc.  But perhaps that's worthwhile given the stars, etc, from the original repo?

What does everyone think?

**Comment by akuker** [comment]
sounds like a good plan to me!

**Comment by da4089** [comment]
@rsb-23 @return42 any feedback on this?

**Reply by rsb-23** [reply]
This seems the best approach to me. 
Additionally, a warning should be added to update/refresh active forks on py-vobject/vobject preferably in contributor.md

- Which release will be merged in step 3, v0.9.7 or 0.9.8?

**Reply by da4089** [reply]
I think it makes most sense to merge the latest code back?
